scripts/components/heading-level/heading-level.js

```diff
@@ -3,6 +3,13 @@ import { __, sprintf } from '@wordpress/i18n';
 import { ToolbarGroup, Path, SVG } from '@wordpress/components';
 import { range } from 'lodash';
 
+/**
+ * A heading level selector option.
```

Comment:
This is confusing me a bit, the doc suggests that this is the selector option, but the name of the component is `HeadingLevelIcon`.

Reply:
It's an option inside the heading level selector 😄  but I can see how it can be confusing. Didn't want to change the name of the component itself for breaking-changes reason, but I'll cahnge the comment to say `HeadingLevelIcon`